int64 timestamp
int8 box_type
float64[8] box
int8 attribute
float64 longitude
float64 latitude
int16 image_cols
int16 image_rows
sensor_msgs/Image image_data

